### PR TITLE
LoggedOut Homepage Feature

### PR DESCRIPTION
**Pull request by samuelsandoval1**
## Proposed Changes
Implement style feature to hide sign up link only when the user is loggedIn. 
[Feature Image](https://screenshot.googleplex.com/OvMTvGkhmER.png)

## Types of Changes
What types of changes does your code introduce to Techish Project?
_Put an x in the boxes that apply_

- [ ]  Bugfix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation Update (if none of the other choices apply)

## Checklist
_Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  I have added necessary documentation (if appropriate)
- [ ]  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/webapp/ts/login.js [hunk 1]
```diff
@@ -37,27 +37,32 @@ var __generator = (this && this.__generator) || function (thisArg, body) {
...
+                        console.log("Inside if statement");
                         if (logStatus.loginUrl === "") {
                             link.setAttribute('href', logStatus.logoutUrl);
                             link.innerHTML = 'Logout';
+                            signUpLink.setAttribute('style', 'display:none;');
                         }
                         else {
                             link.setAttribute('href', logStatus.loginUrl);
                             link.innerHTML = 'Login';
+                            console.log("HREF: " + logStatus.loginUrl);
```
Comment (hadleyshapland) [inline src/main/webapp/ts/login.js:65]: Delete?

File: src/main/webapp/ts/login.js [hunk 2]
```diff
@@ -76,6 +81,7 @@ function logStatusMethod() {
                     return [4 /*yield*/, response.json()];
                 case 2:
                     currentStatus = _a.sent();
+                    console.log(currentStatus);
```
Comment (hadleyshapland) [inline src/main/webapp/ts/login.js:84]: Delete?

Comment (samuelsandoval1) [reply src/main/webapp/ts/login.js]: Done

File: src/main/webapp/ts/login.js [hunk 3]
```diff
@@ -37,27 +37,32 @@ var __generator = (this && this.__generator) || function (thisArg, body) {
...
                 case 1:
                     logStatus = _a.sent();
                     if (logStatus.hasProfile == false && logStatus.loginUrl == "") {
                         document.location.href = "/profileBuilder.html";
                     }
+                    console.log("HREF before if: " + logStatus.loginUrl);
                     //set up function to set login/logout link based on which string is non empty
-                    if (link && logging) {
+                    if (link && logging && signUpLink) {
+                        console.log("Inside if statement");
```
[the remark this answers is not in the document]
Comment (samuelsandoval1) [reply src/main/webapp/ts/login.js]: Done

File: src/main/webapp/ts/login.js [hunk 4]
```diff
@@ -37,27 +37,32 @@ var __generator = (this && this.__generator) || function (thisArg, body) {
...
                     logging = document.getElementById('navbarResponsive');
                     link = document.getElementById('login-link');
+                    signUpLink = document.getElementById('signup-link');
                     return [4 /*yield*/, logStatusMethod()];
                 case 1:
                     logStatus = _a.sent();
                     if (logStatus.hasProfile == false && logStatus.loginUrl == "") {
                         document.location.href = "/profileBuilder.html";
                     }
+                    console.log("HREF before if: " + logStatus.loginUrl);
```
[the remark this answers is not in the document]
Comment (samuelsandoval1) [reply src/main/webapp/ts/login.js]: Done

**Pull request merged**